playwright-tests/fork-snippet.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from "@playwright/test"
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("http://127.0.0.1:5173/seveibar/push-button")
```

Comment:
If you look at the error logs, you are using the port number `5173` while the localhost uses the `5177`.

![image](https://github.com/user-attachments/assets/dac33c53-301e-4534-af4d-fcc8eda44181)